### DISCUSSION TITLE
How does eQSL work anyway?

### DISCUSSION THREAD

**Discussion by MikleBas** [magicbug/Cloudlog#1778] [Q&A]
1. I can’t understand, do I need to manually upload the log to the service or is it automatically uploaded there?
2. Due to the fact that there are restrictions on the execution time of scripts, the download of links from your program is terminated by a time limit. Is it possible to assign this task to cron?

**Comment by ofadam** [comment]
Is this similar to what you're experiencing? 

https://github.com/magicbug/Cloudlog/issues/1635

**Reply by MikleBas** [reply]
No, everything works stably not fast for me on both versions of PHP
PS I only use 1 location and 1 nickname

**Reply by MikleBas** [reply]
> Is this similar to what you're experiencing?
> 
> #1635

I don’t automatically download anything anywhere, I just do everything manually :)
then what is the program for? :)
